### ISSUE TITLE
Accessibility : MAS05A : Keyboard : Focus is not visible on the actively selected element in R History window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/RTVS#3338]
Platform:
Windows 10 RS1
Visual Studio Enterprise 2017 Preview
Version: 15.0.26340.0

Repro Steps:
1. Launch Visual Studio
2. Hit "Alt+R", to open "Rtools" menu
3. Using arrow keys navigate to "Data Science Settings" menu item and hit enter
4. In the "R Interactive" pane, enter the command "x<-iris" and hit enter to get the variable in the "Variable Explorer"
5. Using Control + Tab, navigate to 'R History' window
6. Navigate to 'Toggle Multiline Selection' button and select it
7. Observe the focus behavior on selecting the button

Expected Result:
Focus should be clear and visible

Actual Result:
When the toggle button is in actively selected state focus is not visible

User Impact:
Motor impaired and cognitively impaired users could not identify the focused element on the window

![kb_05a](https://cloud.githubusercontent.com/assets/22956130/24294953/ac7763f2-10be-11e7-9b0a-24f4d4f53df4.png)

**Comment by MikhailArkhipov** [comment]
This is VS IDE issue. opened VSTS 399981
